frontend/src/container/withActivity.jsx

```diff
@@ -69,6 +72,11 @@ const withActivity = (WrappedComponent, setActivityList, setActivityNextPage, re
       this.changingActivityList = true
       const updatedActivityList = await addMessageToActivityList(data, props.activity.list, FETCH_CONFIG.apiUrl)
       props.dispatch(setActivityList(updatedActivityList))
+      const showRefresh = (
+        updatedActivityList.length > 0 &&
+        updatedActivityList[0].newestMessage.event_id !== data.event_id
```

Comment:
Is updatedActivityList[0].newestMessage always defined?

Reply:
As long as there is an activity in the list (tested just above) it is, an activity always has a `newestMessage`, either coming from the messages API or a TLM.